src/main/features/core/tray.js

```diff
@@ -138,7 +142,9 @@ Emitter.on('audiooutput:list', (event, devices) => {
       });
     }
   });
-  setContextMenu();
+  if (appIcon) {
```

Comment:
Why do we have like 3 if statements in the same file...  Why can't all the code be inside one if statement